autonomic-algorithms-commons/src/main/java/br/com/autonomiccs/autonomic/algorithms/commons/resources/HostResources.java

```diff
@@ -28,7 +28,7 @@
 
 public class HostResources implements Cloneable {
 
-    private List<VmResources> vmsResources = new ArrayList<VmResources>();
+    private List<VmResources> vmsResources = new ArrayList<>();
     private float cpuOverprovisioning, memoryOverprovisioning;
```

Comment:
@rafaelweingartner I don't understand why you remove the type from the constructor, both are correct I know but in older versions of java (6 and older) , the type in a list constructor is necessary, at the last this code won't compile in java 6 or older versions.